HLTrigger/special/plugins/HLTMuonRecHitClusterFilter.cc

```diff
@@ -61,13 +69,17 @@ void HLTMuonRecHitClusterFilter::fillDescriptions(edm::ConfigurationDescriptions
   desc.add<edm::InputTag>("ClusterTag", edm::InputTag("hltCSCrechitClusters"));
   desc.add<int>("MinN", 1);
   desc.add<int>("MinSize", 50);
-  desc.add<int>("MinSizeMinusMB1", 0);
-  desc.add<int>("Max_nMB1", 0);
-  desc.add<int>("Max_nMB2", 0);
-  desc.add<int>("Max_nME11", 0);
-  desc.add<int>("Max_nME12", 0);
-  desc.add<int>("Max_nME41", 0);
-  desc.add<int>("Max_nME42", 0);
+  desc.add<int>("MinSizeMinusMB1", 999);
+  desc.add<int>("MinSizeEtaLT1p9NstationGT1", 999);
+  desc.add<int>("MinSizeEtaLT1p9NstationEq1", 999);
+  desc.add<int>("MinSizeEtaGT1p9NstationGT1", 999);
+  desc.add<int>("MinSizeEtaGT1p9NstationEq1", 999);
+  desc.add<int>("Max_nMB1", 999);
+  desc.add<int>("Max_nMB2", 999);
+  desc.add<int>("Max_nME11", 999);
+  desc.add<int>("Max_nME12", 999);
+  desc.add<int>("Max_nME41", 999);
+  desc.add<int>("Max_nME42", 999);
```

Comment:
The filter requires to pass one of the size requirements (`MinSize`, `MinSizeMinusMB1`, or `MinSizeEta*`) and all of the additional selections (e.g. station vetos). 

Therefore we updated the default values for the additional selections to be always passing, and the other two size requirements (`MinSizeMinusMB1`, or `MinSizeEta*`) are defaulted to fail

Reply:
> Therefore we updated the default values for the additional selections to be always passing, and the other two size requirements (MinSizeMinusMB1, or MinSizeEta*) are defaulted to fail

I'm not sure I follow the rationale. Why not just use as defaults values that are considered reasonable/most likely for these cuts? Let's see what Martin G. suggests/prefers.

Reply:
The change for the `MinSize` part is necessary, because the logic of the filter changed. For example, when we want to require eta-dependent size cut with `MinSizeEta*`, we do not want the filter to pass just by passing `MinSize`.
If we have the `clusterType` parameter, we could simplify this. 

The `Max_n*` can have their sensible default values, which would be 
```
  desc.add<int>("Max_nME11", 0 );
  desc.add<int>("Max_nME12", 0);
  desc.add<int>("Max_nME41", 999);
  desc.add<int>("Max_nME42", 999);
  desc.add<int>("Max_nMB1", 0);
  desc.add<int>("Max_nMB2", 999);
```
It was an oversight that `nME41/nME42/nMB2` are defaulted to `0` previously.